src/views/social/Google.js

```diff
@@ -0,0 +1,10 @@
+import '../../assets/css/socialLogin.scss';
+import { GOOGLE_CALLBACK_URL } from '../../config/index';
+import googleIcon from '../../assets/images/google_32px.png';
+import socialLoginButton from '../../components/socialLoginButton';
+
+const Google = () => (
+  socialLoginButton(GOOGLE_CALLBACK_URL, googleIcon, 'log in with Google', 'Signup with Google')
+);
+
```

Comment:
I would suggest making this component reusable, bypassing props as an argument so that Hezron who is working on the signup page and me on the login page, we could use it instead of passing `log in with Google, Signup with Google`.

Reply:
Note that you will be require to pass name of button as props.